lib/client.js

```diff
@@ -202,6 +242,17 @@ Client.prototype._invoke = function(method, args, location, callback, options, e
     },
     xmlnsSoap = "xmlns:" + envelopeKey + "=\"http://schemas.xmlsoap.org/soap/envelope/\"";
 
+  if(name === 'pullFile') {
```

Comment:
again, is `pullFile` a specified element of the WSDL specification or "just" related to your project?